### ISSUE TITLE
API - Mime type for calendar invite

### ISSUE THREAD

**Issue by kevin-olbrich** [zammad/zammad#1919]
### Infos:

* Used Zammad version: 2.4.0, daily / latest
* Installation method (source, package, ..): package / repo
* Operating system: CentOS 7
* Database + version: -
* Elasticsearch version: 5.x
* Browser + version: -

### Expected behavior:

* Setting ["mime-type": "text/calendar;method=REQUEST"] should keep the mime-type

### Actual behavior:

* Setting ["mime-type": "text/calendar;method=REQUEST"] results in:

```
----==_mimepart_5ac4bc01d4a2b_4ec52ae183bdb5e8196795
Content-Type: application/octet-stream; filename="invite.ics"
Content-Transfer-Encoding: 7bit
Content-Disposition: attachment; filename="invite.ics"
Content-ID: <5ac4bc01d7ba4_4ec52ae183bdb5e81968dc@example.com.mail>

...content...

----==_mimepart_5ac4bc01d4a2b_4ec52ae183bdb5e8196795--
```

### Steps to reproduce the behavior:

* Send API Request (docs: https://docs.zammad.org/en/latest/api-ticket-article.html) with attachment

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by thorsteneckel** [comment]
Hi @kevin-olbrich, could you please post an example request of yours?

**Comment by kevin-olbrich** [comment]
Sure, here we go:

URL: "POST /api/v1/ticket_articles/ HTTP/1.1" 201 None

JSON:
```
{
    "attachments": [
        {
            "data": "base64-encoded-string",
            "filename": "invite.ics",
            "mime-type": "text/calendar;method=REQUEST"
        }
    ],
    "body": "body-text",
    "content_type": "text/plain",
    "internal": false,
    "subject": "Test - Calendar (Calendar-ID 1234)",
    "ticket_id": "8209",
    "to": "ko@example.com",
    "type": "email"
}
```

**Comment by kevin-olbrich** [comment]
@muhammadn 
I think I need to explain a little more, as the case you have tested, ist not exactly what I have in mind.

Attaching an ics file in my email article is working perfectly (your test case is fine).
What I tried to do, is sending this article as a calendar invitation like you do on Microsoft Exchange.
These e-mails are then shown as calendar invites in Outlook inbox, Google calendar, iPhone, etc.
Using the ics as attachment shows as regular email. Most clients then show that this email also includes an appointment.

An example how to do it in python is located here:
https://stackoverflow.com/questions/4823574/sending-meeting-invitations-with-python

Maybe I got the mime part wrong but zammad does not honor the mime type I specified.
